packages/mobile/src/storage/keychain.tsx

```diff
@@ -40,3 +40,14 @@ export async function retrieveStoredItem(key: string) {
     throw new Error(ErrorMessages.KEYCHAIN_STORAGE_ERROR)
   }
 }
+
+export async function removeStoredItem(key: string) {
+  try {
+    return await Keychain.resetGenericPassword({
```

Comment:
In general, I'd recommend against `return await`. You may as well just return the promise directly and have the caller await instead of making this function async